keepsorted/block.go

```diff
@@ -53,22 +55,29 @@ const (
 // include is a function that lets the caller determine if a particular block
 // should be included in the result. Mostly useful for filtering keep-sorted
 // blocks to just the ones that were modified by the currently CL.
-func (f *Fixer) newBlocks(lines []string, offset int, include func(start, end int) bool) (blocks []block, incompleteBlocks []incompleteBlock) {
-	startIndex := -1
-	startLine := ""
+func (f *Fixer) newBlocks(lines []string, offset int, include func(start, end int) bool) ([]block, []incompleteBlock) {
```

Comment:
Is this worth a documentation update? Or is this something that might be reasonable to let users discover on their own and be extremely happy that it "just works"?